scripts/newinstall.sh

```diff
@@ -165,7 +165,7 @@ fi
 ##########	Test/warn about Python versions, offer to get miniconda2 if too old
 
 if true; then
-	PYVEROK=$(python -c 'import sys; print("%i" % (sys.hexversion >= 0x02070000 and sys.hexversion < 0x03000000))')
+	PYVEROK=$($PYTHON -c 'import sys; print("%i" % (sys.hexversion >= 0x02070000 and sys.hexversion < 0x03000000))')
```

Comment:
I'm not sure how to reword it -- it's clear to me as-is.  I have no objection to you updating it to suite your taste.